gallery_dl/extractor/instagram.py

```diff
@@ -319,3 +383,18 @@ def get_metadata(self):
     def instagrams(self):
         url = '{}/explore/tags/{}/'.format(self.root, self.tag)
         return self._extract_tagpage(url)
+
+
+class InstagramStoriesExtractor(InstagramExtractor):
+    """Extractor for StoriesPage"""
+    subcategory = "stories"
+    pattern = (r"(?:https?://)?(?:www\.)?instagram\.com"
+               r"/stories/([^/?&#]+)")
+
```

Comment:
You should  add an "only-matching" test, even if regular tests aren't possible. Something like
```
test = ("<url>",)
```

Reply:
Thanks, I have added it!